definitions/pipelines/somatic_exome.cwl

```diff
@@ -124,13 +124,25 @@ inputs:
           In general for a WES exome reagent bait_intervals and target_intervals are the same.
     per_base_intervals:
         type: ../types/labelled_file.yml#labelled_file[]
+        label: "per_base_intervals: yaml file specifying name and input for picard CollectHsMetrics"
+        doc: |
+          Optional input, redundant with bait_intervals, provides a way to name name the output file from running picard CollectHsMetrics. See ../types/labelled_file.yml for an example.
```

Comment:
In `subworkflows/hs_metrics.cwl`, the CollectHsMetrics tool runs multiple times with different options.  It's not just a different filename; it's a different mode of running!

Reply:
Perhaps this would help: https://github.com/genome/analysis-workflows/wiki/Frequently-Asked-Questions-(FAQ)#somatic-exome